cmd/kube-controller-manager/controller-manager.go

```diff
@@ -49,6 +49,7 @@ var (
 	cloudProvider   = flag.String("cloud_provider", "", "The provider for cloud services.  Empty string for no provider.")
 	cloudConfigFile = flag.String("cloud_config", "", "The path to the cloud provider configuration file.  Empty string for no configuration file.")
 	minionRegexp    = flag.String("minion_regexp", "", "If non empty, and -cloud_provider is specified, a regular expression for matching minion VMs.")
+	nodeSyncPeriod  = flag.Duration("node_sync_period", 10*time.Second, "The period for syncing nodes from cloudprovider.")
```

Comment:
How about adding this to flag docstring:
`Longer periods will result in fewer calls to cloud provider, but may delay addition of new nodes to cluster.`